epoxy-adapter/src/main/java/com/airbnb/epoxy/ModelGroupHolder.kt

```diff
@@ -217,7 +210,12 @@ private class ViewStubData(
         }
     }
 
-    fun removeView() {
+    fun resetStub() {
```

Comment:
`private` ?

Reply:
it's actually used in the holder class in the same file, but ViewStubData is private

Reply:
ah, could reduce to `internal` then ;)